pages/client-libraries/java.mdx

```diff
@@ -4,105 +4,687 @@ description: Peruse our guide on implementing Java in Memgraph. Comprehensive do
+            return result.list();
+        }
+
+    }
+
+    public static void main(String[] args) {
+
+        try (var helloMemgraph = new HelloMemgraph("bolt://localhost:7687", "", "")){
+            // Create indexes
+            helloMemgraph.createIndexes();
+            // Create nodes and relationships
+            helloMemgraph.createNodesAndRealationships();
+            // Read nodes
+            var result = helloMemgraph.readNodes();
+            // Process results
+            for (Record node : result) {
+                Node n = node.get("n").asNode();
+                System.out.println(n); // Node type
+                System.out.println(n.asMap()); // Node properties
+                System.out.println(n.id()); // Node internal ID
+                System.out.println(n.labels()); // Node labels
+                System.out.println(n.get("id").asLong()); // Node user defined id property
+                System.out.println(n.get("name").asString()); // Node user defined name property
+                System.out.println(n.get("description").asString()); // Node user defined description property
+            }
+        }
+    }
+```
+This code snippet is a minimal working example that will create a connection to the Memgraph database via Java driver, create indexes, create nodes and relationships, and read the data back. Take a look at the code in detail as
+it includes the comments to help you understand it. More details about the code snippet will be explained in the [Java client API usage and examples](#java-client-api-usage-and-examples) section.
+
+
+### Run the application
+
+Once code is in place, you should be able to run the application. If you are using an IDE, you can run the application from there. If everything is working properly, you should see the following output:
+
+```text
+node<76>
+{name=Memgraph, createdAt=1693389566657284, description=Fastest graph DB in the world!, id=1}
+76
+[Technology]
+1
+Memgraph
+Fastest graph DB in the world!
+```
+
+### Visualize the data 
+To visualize objects created in the database with the the code snippet above, head over to [http://localhost:3000/](http://localhost:3000/) and run `MATCH path=(n)-[p]-(m) RETURN path` in the *Query Execution* tab. 
+That query will visualize the created nodes and relationships. By clicking on a node or relationship, you can explore different properties.
+
+![java-quick-start](/pages/client-libraries/java/java-quick-start.png)
+
+### Next steps 
+
+This makes a good starting point for Java application. For more information on how to use the Java driver, 
+continue reading the guide under [Java client API usage and examples](#Java-client-api-usage-and-examples). 
+
+</Steps>
+
+<Callout type="info">
+Memgraph created [Bolt Java Driver](https://github.com/memgraph/bolt-java-driver) which can be used to connect to a running Memgraph instance. But we still recommend you use the above mentioned driver.
+</Callout> 
```

Comment:
```suggestion
<Callout type="info">
Memgraph created [Bolt Java Driver](https://github.com/memgraph/bolt-java-driver) which can be used to connect to a running Memgraph instance. But we still recommend you use the above mentioned driver.
</Callout> 
```

Is it necessary to mention this twice? If yes, update the language with my corrections

Reply:
```suggestion
```
Sorry, it should be deleted here.